python/lsst/afw/table/Catalog.i

```diff
@@ -255,7 +255,8 @@ public:
             # self._columns is created the when self.columns is accessed -
             # looking for it in self.columns below would trigger infinite
             # recursion.
-            if name == "_columns":
+            # Test for "this" as a temporary workaround for ticktet 7377
```

Comment:
Highly likely given this code in the `.py` file:
```python
def _swig_setattr_nondynamic(self, class_type, name, value, static=1):
    if (name == "thisown"):
        return self.this.own(value)
    if (name == "this"):
        if type(value).__name__ == 'SwigPyObject':
            self.__dict__[name] = value
            return
```
so SWIG is using a "this" attribute.